src/utilities/roundNumberToDecimalPlaces.ts

```diff
@@ -0,0 +1,7 @@
+// precise rounding https://www.jacklmoore.com/notes/rounding-in-javascript/
```

Comment:
That context on why toFixed is really useful and I'd hate for it to get lost to history. Lets put it in as a comment.

```suggestion
// Because everything is a float in JS, Number.toFixed sometimes rounds in the
// "wrong" direction because of float imprecision. For instance:
// `(1.005).toFixed(2)` is `1.00`, NOT `1.01` because 1.005 in floating point is
// actually 1.004999995. By using exponentiation tricks here we can work around
// this imprecision, so `roundNumberToDecimalPlaces(1.005)` returns the expected
// value of `1.01`
// See https://www.jacklmoore.com/notes/rounding-in-javascript/
```